acquia_lift.admin.inc

```diff
@@ -938,15 +997,13 @@ function acquia_lift_update_targeting($agent) {
  *   The name of hte targeting audience to assign the option to.
  * @throws \PersonalizeException
  */
-function acquia_lift_set_option_for_audience($agent, $option_id, $audience) {
-  $option_set = acquia_lift_get_option_set_for_targeting($agent->machine_name);
+function acquia_lift_set_option_for_audience(&$option_set, $option_id, $audience) {
```

Comment:
Docblock updates for all of these.....
I feel like I'm going to earn some sort of docblock nag title :-)